BUILD

```diff
@@ -1,5 +1,35 @@
 python_requirements(
-    name="root",
+    name="reqs",
+    source="requirements-pants.txt",
+    module_mapping={
```

Comment:
Maybe link to your PR to upstream these, so future editors can check if this can be deleted